cinema/tests/test_movie_api.py

```diff
@@ -70,8 +70,8 @@ def test_post_invalid_movies(self):
             },
         )
         superman_movies = Movie.objects.filter(title="Superman")
-        self.assertEqual(movies.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(superman_movies.count(), 0)
+        self.assertEqual(movies.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(superman_movies.count(), 1)
```

Comment:
You are not supposed to change the tests. Instead, the `actors` and `genres` fields should be required.

Reply:
This comment was among the most important about the whole review. I'd suggest fixing it after all. :)